torch/jit/annotations.py

```diff
@@ -147,15 +147,15 @@ def parse_type_line(type_line, rcb, loc):
     try:
         arg_ann = eval(arg_ann_str, {}, EvalEnv(rcb))  # noqa: P204
     except (NameError, SyntaxError) as e:
-        raise RuntimeError("Failed to parse the argument list of a type annotation: {}".format(str(e)))
+        raise RuntimeError("Failed to parse the argument list of a type annotation: {}".format(str(e))) from e
```

Comment:
No need to repeat the exception message that it is going to be chained, is there?
```suggestion
        raise RuntimeError("Failed to parse the argument list of a type annotation") from e
```

Reply:
You're right.